core/templates/pages/splash-page/splash-page.component.html

```diff
@@ -432,22 +432,37 @@ <h2 [innerHTML]="'I18N_SPLASH_VOLUNTEERS_TITLE' | translate"></h2>
     color: #6d6d6d;
   }
 
-  .splash-page .oppia-splash-section-four-left h3,
+  .splash-page .oppia-splash-section-four-left h2,
   .splash-page .oppia-splash-section-five-left h3,
-  .splash-page .oppia-splash-section-six-right h3,
-  .splash-page .oppia-splash-section-seven-left h3 {
+  .splash-page .oppia-splash-section-six-right h2,
+  .splash-page .oppia-splash-section-seven-left h2 {
     color: #6d6d6d;
   }
 
   .splash-page .oppia-splash-section-four-left h2,
-  .splash-page .oppia-splash-section-five-left h2,
   .splash-page .oppia-splash-section-six-right h2,
   .splash-page .oppia-splash-section-seven-left h2 {
+    font-size: 1.17em;
+    font-weight: bold;
+    line-height: 1.1;
+  }
+
+  .splash-page .oppia-splash-section-four-left h3,
+  .splash-page .oppia-splash-section-five-left h2,
```

Comment:
Why is it still `h2` in section five?

Reply:
Hmm, section five is weird it seems to be used for the student quotes there. Semantically that doesn't really make sense, can we transform it to spans or paragraphs with some class?

Reply:
Section five actually has the h2 and h3 tags in the correct order.

Reply:
I definitely can transform it to some span or paragraph if you want.

Reply:
Yeah, transform it

Reply:
I've made them into paragraph tags with suitable class names. That should solve it.

Reply:
I've updated the template to reflect the changes in that section too.